src/chart.js

```diff
@@ -69,12 +68,11 @@ export async function createChart(container) {
       symbolSize: +d.symbol_size,
       strokeWidth: +d["stroke-width"],
     }));
-
     const measureData = measureTable.objects();
-    const measures = measureData.map((d) => ({
-      measure: d.measure,
-      value: +d.value,
-    }));
+    const measures = {};
+    measureData.forEach((d) => {
+      measures[d.measure] = +d.value;
+    });
 
```

Comment:
```suggestion
    const measures = measureTable
      .objects()
      .reduce((acc, d) => ({ [d.measure]: d.value, ...acc }));

```

Как-будто функция `getMeasureValue` тогда и не нужна.